packages/discovery-provider/src/api/v1/models/tracks.py

```diff
@@ -139,11 +152,20 @@
         "cover_art": fields.String,
         "remix_of": fields.Nested(full_remix_parent),
         "is_available": fields.Boolean,
-        "is_premium": fields.Boolean,
-        "premium_conditions": fields.Raw(allow_null=True),
+        # the following "premium" fields are deprecated and will be removed in the future
```

Comment:
ok cool, so we're keeping these fields a bit longer

Reply:
yup